app/src/main/res/values/strings.xml

```diff
@@ -628,6 +630,10 @@
     <string name="nc_expire_message_one_hour">1 hour</string>
     <string name="nc_expire_messages_explanation">Chat messages can be expired after a certain time. Note: Files shared in chat will not be deleted for the owner, but will no longer be shared in the conversation.</string>
 
+    <!-- Breakout rooms -->
+    <string name="switch_to_main_room">Switch to main room</string>
```

Comment:
Not sure about "main room", but can adjust later if we use a different wording in Web/iOS